redirects.go

```diff
@@ -158,5 +158,23 @@ func parseStatus(s string) (code int, err error) {
 	}
 
 	code, err = strconv.Atoi(s)
+	if err != nil {
+		return code, err
```

Comment:
nit: no value if an error occurred, removes surface for bugs when err is ignored
```suggestion
		return nil, err
```

Reply:
I can't return nil because it's an int.  I'll return 0 though.